pkg/spanconfig/spanconfigreconciler/reconciler.go

```diff
@@ -461,6 +461,10 @@ func updateSpanConfigRecords(
 			}
 			return err // not a retryable error, bubble up
 		}
+
+		if log.V(2) {
+			log.Infof(ctx, "successfully updated span config records: deleted = %+#v; upserted = %+#v", toDelete, toUpsert)
+		}
```

Comment:
Even at V(2) I wonder if we want this particular log message.   It's rather chatty and hard to read. I leave it to your judgement.